### PR TITLE
Enhancement/select design component

### PR DESCRIPTION
**Pull request by brunomichetti**
- React Select separated in component
- React Select Style
- Improve style of button and input in user form: shadow and borders

Screen Shot 2020-06-30 at 14 28 19

Screen Shot 2020-07-01 at 13 54 37

Screen Shot 2020-06-30 at 14 28 42

Screen Shot 2020-06-30 at 14 28 49

### CODE REVIEW

File: src/components/users/SignUpForm.js [hunk 1]
```diff
@@ -89,6 +91,7 @@ const SignUpForm = () => {
         inputName="password1"
         inputValue={password1}
         inputOnChange={handleChange}
+        inputPlaceHolder="MIN. 8 CHARACTERS LONG"
```
Comment (tamarasuarezrod) [inline src/components/users/SignUpForm.js]: get this from `en.js`

File: src/components/users/UserSelect.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Select from 'react-select';
+import { FormattedMessage } from 'react-intl';
+
+import './user-form.scss';
+import { black } from 'color-name';
+
+const customStyle = {
```
Comment (tamarasuarezrod) [inline src/components/users/UserSelect.js]: shouldn't this be in a style file? 🤔

File: src/components/users/SignUpForm.js (hunk 1 above)
Comment (brunomichetti) [reply src/components/users/SignUpForm.js]: If I put `<FormattedMessage id="userform.password.placeholder.label.text" />` then I don't see the message. I see something like `object.Object`. I need to put a raw string there. Do you know why?

File: src/components/users/UserSelect.js (hunk 2 above)
Comment (brunomichetti) [reply src/components/users/UserSelect.js]: Are you saying to put this in a scss file?

Comment (tamarasuarezrod) [reply src/components/users/UserSelect.js]: yes

File: src/components/users/SignUpForm.js (hunk 1 above)
Comment (tamarasuarezrod) [reply src/components/users/SignUpForm.js]: I think it should be something like: `placeholder={intl.formatMessage({ id: 'generic.to' })}`

**Review by tamarasuarezrod — APPROVED** [review]

**Review by manema — APPROVED** [review]

**Pull request merged**